doc/api/url.md

```diff
@@ -1076,17 +1107,33 @@ added: v10.12.0
 This function ensures that `path` is resolved absolutely, and that the URL
 control characters are correctly encoded when converting into a File URL.
 
-```js
-new URL(__filename);                // Incorrect: throws (POSIX)
-new URL(__filename);                // Incorrect: C:\... (Windows)
-pathToFileURL(__filename);          // Correct:   file:///... (POSIX)
-pathToFileURL(__filename);          // Correct:   file:///C:/... (Windows)
+```mjs
+import {pathToFileURL, fileURLToPath} from 'url';
+const filename = fileURLToPath(import.meta.url);
+new URL(filename);                    // Incorrect: throws (POSIX)
+new URL(filename);                    // Incorrect: C:\... (Windows)
+pathToFileURL(filename);              // Correct:   file:///... (POSIX)
+pathToFileURL(filename);              // Correct:   file:///C:/... (Windows)
```

Comment:
I still think this seems a bit pointless (why would anyone transform `import.meta.url` to a path and try to use it as a URL afterwards?).
```suggestion
```